examples/land-cover-map/SI_LULC_pipeline.ipynb

```diff
@@ -2060,6 +2060,11 @@
    "toc_position": {},
    "toc_section_display": true,
    "toc_window_display": false
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "24e961c4fcc1e9d399957c48f9f25913c6ada002d310cd99a7e82976b5eba656"
+   }
```

Comment:
Let's see if we can remove this straight from GitHub (cause it's annoying to do in IDE)
```suggestion
```